lib/managers/files.js

```diff
@@ -3,6 +3,14 @@
  */
 
 'use strict';
+// -----------------------------------------------------------------------------
+// Typedefs
+// -----------------------------------------------------------------------------
+
+/**
+ * A webhook trigger type constant
+ * @typedef {FileRepresentationType} Different representations we can request from reps endpoint
```

Comment:
The correct format here is `@typedef {string} FileRepresentationType` — see http://usejsdoc.org/tags-typedef.html